README.md

```diff
@@ -171,4 +172,51 @@ class Entity extends ActiveRecord
     ...
 }
 
+```
+
+or
+
+```php
+<?php
+
+...
+
+class Entity extends ActiveRecord
+{
+    ...
+    
+    public function rules()
+    {
+        return [
+            [['many_to_many_attribute'], 'safe']
+        ];
+    }
+
+    public function getMany_to_many_attribute()
+    {
+        return $this->hasMany(ManyToManyModel::className(), 
+            ['id' => 'many_to_many_model_id'])
+                ->viaTable('table_many_to_many', ['entity_id' => 'id']);
```

Comment:
I use such relation


```php
<?php
return $this->hasMany(Woo::className(), ['id' => 'woo_id'])
   ->viaTable(WooRelation::tableName(), ['item_id' => 'id'], function ($query) {
     /** @var ActiveQuery $query */
     return $query->onCondition(['item_type' => 'woo']);
   }
);
```

and it's didn't work fine - old tags removes after save, remains only new tags.

If problem in RelationBehavior, you shoud fix it, else just add correct example that will works.